src/types.ts

```diff
@@ -180,12 +191,15 @@ export type WsAuthenticatedUserData = {
   eventEmitter: Emitter<IUWebSocketEventMap>
   auth: true
   address: string
+  clientId: string
```

Comment:
Just curious, what's client id on this context and also on `WsNotAuthenticatedUserData`

Reply:
Renamed to `wsConnectionId`